ros/src/computing/perception/detection/visualizers/packages/integrated_viewer/node/data_rate_checker_plugin/data_rate_checker_plugin.h

```diff
@@ -56,17 +55,15 @@ namespace integrated_viewer
 
     // The ROS subscriber
     ros::Subscriber topic_sub_;
+
     // The ROS timer
     ros::WallTimer timer_;
 
     // Variables for message timing
-    unsigned int window_size_ = 10;
-
-    unsigned int message_count_ = 0;
+    unsigned int window_size_ = 10000;
+    double last_message_time_ = -1;
+    double last_printed_time_ = -1;
```

Comment:
This variable doesn't appear to be used.

Reply:
Well spotted. I have removed it, thanks.